ksql-engine/src/test/java/io/confluent/ksql/function/InternalFunctionRegistryTest.java

```diff
@@ -94,7 +94,12 @@ public void shouldCreateCopyOfRegistry() {
 
     assertThat(copy.getUdfFactory("func").getFunction(Collections.emptyList()), equalTo(func));
     assertThat(copy.getUdfFactory("func2").getFunction(Collections.emptyList()), equalTo(func2));
-    assertThat(functionRegistry.getUdfFactory("func2"), nullValue());
+    try {
```

Comment:
This duplicates `shouldThrowExceptionIfNoFunctionsWithNameExist` testing, right?

Reply:
It does, but proves that the new function doesn't get added to the registry it was copied from. Worth keeping both